### ISSUE TITLE
io/clipboard on linux: command fails if xclip is not installed

### ISSUE THREAD

**Issue by AustinArey** [pskfyi/handy#7]
Maybe we can add (error output includes "File not found")?:include "apt-get install xclip" in error output

![image](https://user-images.githubusercontent.com/92492121/234362060-bfdb0bf6-d926-40bf-92b0-7a6cfe7f9f8c.png)

**Comment by pskfyi** [comment]
That is so interesting, I thought it was installed by default in Ubuntu. We'd have run into this in some other distro eventually though. Good to catch it now.

We should be able to turn this into a more helpful error, yes. I don't think we should recommend `apt` specifically - I believe it's only on Debian-based linux distros. But we can at least inform the user with a more targeted message, hiding some of our implementation details. We do still want to throw an error though, so that it will work as expected with `try {} catch {}` clauses, so unfortunately the stack trace will be unavoidable.

It should be possible to handle this in the following manner if you're interested in investigating:

```ts
// in cmd.ts

export class CmdNotFoundError extends Deno.error.NotFound {
  // look for another custom error implementation in the repo for examples
}

// ...

try {
  const process = Deno.run({ cmd })
  // ...
} catch (error) {
  if (error instanceof Deno.error.NotFound) {
    throw CmdNotFoundError(cmd[0]) // the zeroth item in cmd is "xclip" on Linux, etc. on other OS's
  } else {
    throw error // re-throw all unexpected error types
  }
}

**Comment by pskfyi** [comment]
We discussed and we're declining to implement for now. Addressed in 85910893122adb763fdbc41c542d63740582c6b5
